src/amo/css/AddonReview.scss

```diff
@@ -0,0 +1,52 @@
+@import "~core/css/inc/vars";
+
+.AddonReview {
+  text-align: center;
+
+  h2 {
+    font-size: $font-size-l;
+    margin-bottom: 0.3em;
+  }
+
+  p {
+    margin: auto;
+    margin-bottom: 2em;
+    width: 80%;
+  }
+
+  textarea {
+    padding: 0.7em;
+    border: 2px solid #f1f1f1;
```

Comment:
It might be tough to name this colour but it looks like it's the same as what's used on the button leading to this page. Might be good to put it into the vars file even if it has a bad name so the next person that uses it knows to fix it.